modules/sanger-tol/blobtoolkit/generatecsv/meta.yml

```
@@ -0,0 +1,77 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "BLOBTOOLKIT_GENERATECSV"
+description: Generate a samplesheet csv for sanger-tol/blobtoolkit
+keywords:
+  - blobtoolkit/generatecsv
+  - generatecsv
+  - csv
+  - samplesheet
+tools:
+  - "blobtoolkit_generatecsv":
+      description: "not a real tool!"
+      homepage: "https://i.need.this.to.lint.org"
+      documentation: "https://i.need.this.to.lint.org"
+      tool_dev_url: "https://i.need.this.to.lint.org"
+      doi: no DOI available
+      identifier: ""
```

lol :)
I can see how motivated you are, so let me help you
```suggestion
      description: "A samplesheet generator for the BlobToolKit pipeline"
      homepage: "https://pipelines.tol.sanger.ac.uk/blobtoolkit"
      documentation: "https://pipelines.tol.sanger.ac.uk/blobtoolkit"
      tool_dev_url: "https://pipelines.tol.sanger.ac.uk/blobtoolkit/usage#samplesheet-input"
      doi: 10.5281/zenodo.7949058
      identifier: ""
```

---

Thanks Matthieu, tbf i didnt do it to be rude. I just didn't see it being part of btk as it is like a pre-processor. 

But i guess at the same time it is specific for btk at the minute.